tests/phpunit/unit/InputToEntity/FullWikibaseItemInputTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace MediaWiki\Extension\WikibaseReconcileEdit\Tests\Unit\InputToEntity;
+
+use Deserializers\Deserializer;
+use MediaWiki\Extension\WikibaseReconcileEdit\InputToEntity\FullWikibaseItemInput;
+use MediaWiki\Extension\WikibaseReconcileEdit\MediaWiki\ReconciliationException;
+
+/**
+ * @covers \MediaWiki\Extension\WikibaseReconcileEdit\InputToEntity\FullWikibaseItemInput
+ */
+class FullWikibaseItemInputTest extends \MediaWikiUnitTestCase {
+
+	public function testItemType() {
```

Comment:
I think it would be good to have a second test that calls `getItem()` with a different type (e.g. `[ 'type' => 'property' ]`) instead of an empty array. This could be done with a `@dataProvider`, since the rest of the test would be identical.

Reply:
Good idea. Done.